Port to libnotify-0.7, require it

Didn't add to configure, no point

---
--- a/src/pidgin-libnotify.c
+++ b/src/pidgin-libnotify.c
@@ -286,7 +286,7 @@
 		g_free (tr_body);
 		return;
 	}
-	notification = notify_notification_new (title, tr_body, NULL, NULL);
+	notification = notify_notification_new (title, tr_body, NULL);
 	purple_debug_info (PLUGIN_ID, "notify(), new: "
 					 "title: '%s', body: '%s', buddy: '%s'\n",
 					 title, tr_body, best_name (buddy));
